Backpack/src/main/java/net/skyscanner/backpack/spinner/BpkSpinner.kt

```diff
@@ -59,16 +59,20 @@ open class BpkSpinner @JvmOverloads constructor(
 ) : FrameLayout(context, attrs, defStyleAttr) {
 
   private val colors = arrayOf(
-    R.color.bpkPrimary,
-    R.color.bpkWhite,
-    R.color.bpkSkyGrayTint01
+    R.color.bpkCoreAccent,
```

Comment:
the existing ones should keep the current colours otherwise they won't work the way they're currently used